vunit/ui/results.py

```diff
@@ -26,3 +31,20 @@ def merge_coverage(self, file_name, args=None):
         """
 
         self._simulator_if.merge_coverage(file_name=file_name, args=args)
+
+    def get_report(self):
+        """
+        Get a report (dictionary) of tests: status, time and output path
+        """
+        return {
+            test.name: {
+                "status": test._status.name,  # pylint: disable=protected-access
+                "time": test.time,
+                "path": re.sub(
```

Comment:
Use os. path api for manipulation of paths

Reply:
I'm not sure about it. The point with the regular expression is that no replacement (removal) is done if the path is, for any reason, not a child of output_path. In order to do so with `os.path`, it seems that a helper function needs to be written. This is because, [relpath](https://docs.python.org/3/library/os.path.html#os.path.realpath) will return a relative path, even if the test path is not a subset of output path. Therefore, some combination with [commonprefix](https://docs.python.org/3/library/os.path.html#os.path.commonprefix) is required. However, `commonprefix` is buggy, and a better solution was introduced in Python 3.5: [commonpath](https://docs.python.org/3/library/os.path.html#os.path.commonpath). See https://stackoverflow.com/questions/7287996/python-get-relative-path-from-comparing-two-absolute-paths. Alternatively, in Python 3.4 pathlib was added https://docs.python.org/3/library/pathlib.html.

Overall, I think it is not worth implementing a solution based on `os.path` which is compatible with both Python 2.7 and Python >=3.5. We'd better keep the simple regex based approach and revisit this when Python 2 is deprecated. What do you think?